module/pom.xml

```diff
@@ -30,6 +30,11 @@
             <artifactId>spring-context</artifactId>
             <version>${springVersion}</version>
         </dependency>
+        <dependency>
```

Comment:
If this is required here because of the CompatibilitySecurityContext then I think is best to have that live in core.